modules/test/playwright/tests/FDS/FDSActions.spec.ts

```diff
@@ -0,0 +1,90 @@
+import {expect, mergeTests} from '@playwright/test';
+
+import {FDSPagesTest} from '../../fixtures/FDSPages.fixture';
+
+export const test = mergeTests(
+	FDSPagesTest
+);
+
+test.describe('LPS-186875 Create generic creation button actions', () => {
+
+    test.afterAll(async () => {
```

Comment:
We can get rid of this phase (to make tests faster) if we use random ids for the data we create in the test. 

An option worth exploring, IMO...

Reply:
There's in fact an util method in util.ts to get random integers. I don't like how it is mplemented but we can use it for the time being.

Reply:
when you say "id" you mean database ids or entity names? Case here creates entries named "Data Set Test", so to avoid clashing in the naming, we might want to add a suffix to each and every name, not because it has to be unique, but rather to avoid clicking on item "A" when we want to click on item "B" but we can't distinguish both because the name is the same.

This relies on someone else at a higher level to clean up the DataSets from time to time

Reply:
I included LPS name intentionally to come up a question: how we are going to track when there is new functionality to test? (and I'm not asking for the tool itself - a list, jira, excel, whatever)

I mean, if  we create tests for every LPS, there will be a lot of overlapping test code, but we'll track accurately to which LPS a test fail belongs in order to fix it, and we can be sure that all functionality are cover. On the other hand, if we create tests based on the functionalities, not in LPSs, we'll be not sure of what LPS are causing a fail in tests.

On a daily basis, when one of us complete a LPS, how can we know if there are tests already covering a part of the functionality we just developed (as it surely is a part of a more complex piece of the portal)? It's maybe a question for the long term, but I think is useful keeping this question in mind from right now

Reply:
> when you say "id" you mean database ids or entity names?

I mean any id. The thing that makes entities we create in the DB unique and thus avoids collisions. I agree that not cleaning up has the downside that the DB grows and grows if no one cleans it but since in the CI the server is going to be destroyed and locally we can simply wipe our local DBs as needed, I think it's worth proposing that cleaning strategy (or lack of :sweat_smile: ) since it is faster, simpler and in a way, safer.

Reply:
I have a cleaning phase in mind but nothing implemented.

Something like:

1. generate a frontend ID (using [randomUUID](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID) or [getRandomInt](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/utils/util.ts))
2. Create the data test with the id embebed
3. Create a locator to match the id
4. Clean the data test with the help of the locator

Reply:
But I'm not sure I asked about [here](https://liferay.slack.com/archives/C3JBR21HA/p1704904037500889?thread_ts=1704879554.590159&cid=C3JBR21HA)